app/Models/Photo.php

```diff
@@ -11,7 +11,21 @@ class Photo extends Model
      *
      * @var array
      */
-    protected $fillable = ['event_id', 'signup_id', 'northstar_id', 'file_url', 'edited_file_url', 'caption', 'status'];
+    protected $fillable = ['event_id', 'signup_id', 'northstar_id', 'file_url', 'edited_file_url', 'caption', 'status', 'source', 'remote_addr'];
```

Comment:
This is random, but one thing I've started to try in my Laravel projects is when the `$fillable` array seems like it's gonna have a lot of stuff, I switch to use `$guarded` to just specify the items _not_ mass-assignable, which is usually a much smaller set of items. Feel free to ignore; just a suggestion cause it kind of became a habit to always use `$fillable` just cause that how we do it elsewhere, but doesn't necessarily have to be the default 😉